### PR TITLE
T2B1 new designs

### PR DESCRIPTION
**Pull request by grdddj**
Overarching PR for all the open `T2B1` PRs:
- https://github.com/trezor/trezor-firmware/pull/3119
- https://github.com/trezor/trezor-firmware/pull/3107
- https://github.com/trezor/trezor-firmware/pull/3098
- https://github.com/trezor/trezor-firmware/pull/3096
- https://github.com/trezor/trezor-firmware/pull/3082

Reason:
- seeing all the UI changes in one place
- easier and quicker fixtures recording

### CODE REVIEW

**Comment by Hannsek** [comment]
How this should continue? 
image

**Comment by Hannsek** [comment]
What about this? It should fit.
image

**Comment by grdddj** [comment]
> 



> What about this? It should fit. image

Yeah, good idea, works much better for both models. Done in [a5d9ce0](https://github.com/trezor/trezor-firmware/pull/3135/commits/a5d9ce0346016247c89a8eb5b9b69738a48fbed7)

image


image

**Comment by Hannsek** [comment]
Please change "input" to "enter" and add an article "a".

**Comment by grdddj** [comment]
> Please change "input" to "enter" and add an article "a".

Done in some of the fixups.

Finally all the tests (upgrade, persistence...) are passing, so we are ready to merge when we decide to

**Comment by andrewkozlik** [comment]
The UI changes need more thorough review. I just opened up one test out of curiosity what the SLIP39 process looks like on the T2B1 and I am seeing things like this (which come from earlier PRs):

image
(nonsense)

and like this:

image
image

(what happened to group threshold?)

Some more random observations I stumbled on while scrolling:

image
(missing URL, also "info" is too informal, there is enough room for proper "information")

image
(title covered by dots)

image
("It should be backed up now!" -> "You should create a backup now!")

image
Why change this? Turn on/off sounds silly. Enable/disable is standard terminology. Furthermore, phrasal verbs are more likely to be misunderstood by non-native speakers, e.g. "Enable wipe code protection" or "Set wipe code" are clear and concise.

Finally, either skip articles in a sentence or use articles throughout the sentence ("Enter _a_ share from a different group.").
image

**Comment by Hannsek** [comment]
Thanks for the catch. The first copy should be about unique word lists: 
image

> (what happened to group threshold?)

group thresholds are only for super shamir, right?
We will tackle this later.

> (missing URL, also "info" is too informal, there is enough room for proper "information")

@b-irving what do you think about "information" vs "info"?
@grdddj URL is indeed missing, can you please fix it?

> (title covered by dots)

There are marquee and the header will "bounce" from side to side. We are currently thinking how this can be improved.

> ("It should be backed up now!" -> "You should create a backup now!")

cc @b-irving. We have agreed previously to shift from personal copy to neutral one.

> Why change this? Turn on/off sounds silly. Enable/disable is standard terminology. Furthermore, phrasal verbs are more likely to be misunderstood by non-native speakers, e.g. "Enable wipe code protection" or "Set wipe code" are clear and 
concise.

This is not entirely true. See e.g. https://uxpsychology.substack.com/p/how-to-optimize-button-microcopy. @b-irving may tell you more.

> Finally, either skip articles in a sentence or use articles throughout the sentence ("Enter a share from a different group.").

@b-irving what do you think about that?

**Comment by grdddj** [comment]
> > (missing URL, also "info" is too informal, there is enough room for proper "information")
> 
> @b-irving what do you think about "information" vs "info"? @grdddj URL is indeed missing, can you please fix it?

Fixing the missing URL in https://github.com/trezor/trezor-firmware/pull/3145 - putting the header just on one line (removing the "Super" word)

File: core/embed/rust/src/ui/model_tr/component/share_words.rs [hunk 1]
```diff
@@ -212,27 +167,18 @@ where
...
+            let mut content = String::<50>::new();
             for i in 0..WORDS_PER_PAGE {
                 let index = self.word_index() + i;
                 if index >= self.share_words.len() {
                     break;
                 }
                 let word = &self.share_words[index];
                 let current_line =
-                    build_string!(20, inttostr!(index as u8 + 1), " ", word.as_ref(), "\n");
+                    build_string!(20, inttostr!(index as u8 + 1), ". ", word.as_ref(), "\n");
```
Comment (matejcik) [inline core/embed/rust/src/ui/model_tr/component/share_words.rs:181]: please also use size 50
(we need a separate review of string sizes used. maybe `build_string!` should just not take this as an argument)

File: core/src/storage/recovery.py [hunk 2]
```diff
@@ -10,13 +10,13 @@
 _IN_PROGRESS               = const(0x00)  # bool
 _DRY_RUN                   = const(0x01)  # bool
 _SLIP39_IDENTIFIER         = const(0x03)  # bytes
-_SLIP39_THRESHOLD          = const(0x04)  # int
```
Comment (matejcik) [inline core/src/storage/recovery.py:13]: what happened here?

File: core/src/trezor/ui/layouts/tr/__init__.py [hunk 3]
```diff
@@ -399,27 +399,35 @@ async def confirm_reset_device(
...
+            "recover_device",
+            title,
+            description="It is safe to eject your Trezor anytime and continue later.",
+            verb="CONTINUE",
+            br_code=ButtonRequestType.ProtectCall,
+        )
+
 
-# TODO cleanup @ redesign
 async def confirm_backup() -> bool:
```
Comment (matejcik) [inline core/src/trezor/ui/layouts/tr/__init__.py:412]: missed this in some previous review:
**confirm**_something should raise when the user cancels.

please rename here and in TT to `ask_backup` or `prompt_backup`

File: core/embed/rust/src/ui/component/text/op.rs [hunk 4]
```diff
@@ -238,7 +239,9 @@ impl<T: StringType + Clone> OpTextLayout<T> {
 #[derive(Clone)]
 pub enum Op<T: StringType> {
     /// Render text with current color and font.
-    Text(T),
+    /// Bool signifies whether the text continues or not
+    /// (so we know whether to show ellipsis or not).
```
Comment (matejcik) [inline core/embed/rust/src/ui/component/text/op.rs:243]: see https://github.com/trezor/trezor-firmware/pull/3107#discussion_r1260868635

File: core/src/storage/recovery.py (hunk 2 above)
Comment (grdddj) [reply core/src/storage/recovery.py:13]: This is not used, so I moved it under Deprecated Keys.

Looking at git history, it seems it hasn't been used from the start (or from moving code into this file)

File: core/src/trezor/ui/layouts/tr/__init__.py (hunk 3 above)
Comment (grdddj) [reply core/src/trezor/ui/layouts/tr/__init__.py]: Done in [546199b](https://github.com/trezor/trezor-firmware/pull/3135/commits/546199beaf948ff9e4298cebec355e1b1e5cd959)

File: core/embed/rust/src/ui/component/text/op.rs (hunk 4 above)
Comment (grdddj) [reply core/embed/rust/src/ui/component/text/op.rs]: Done in [6c0fd54](https://github.com/trezor/trezor-firmware/pull/3135/commits/6c0fd547b0a383aa6a1bf480d01e7a8bdd5fc27b)

File: core/embed/rust/src/ui/model_tr/component/share_words.rs (hunk 1 above)
Comment (grdddj) [reply core/embed/rust/src/ui/model_tr/component/share_words.rs]: Done in [d4142d4](https://github.com/trezor/trezor-firmware/pull/3135/commits/d4142d4e9aec4987e54eccb2554311810500d57f)

**Review by matejcik — APPROVED** [review]

**Pull request merged**